### ISSUE TITLE
CollectionViewController with MDCFlexibleHeader can't scroll to top using 3-finger a11y scrolling gesture

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#4133]
A collection view cannot be scrolled completely to the bottom then completely to the top using the 3-finger swiping gesture.  It seems likely that this is occurring because 1) the contentSize height is not an even multiple of the height of the collection view, and 2) MDCFlexibleHeaderView is updating the contentInset to a non-UIEdgeInsetsZero value.  The 3-finger scroll gesture stops when the contentOffset is {0, 0}, so it might just be that this gesture is broken for all scroll views using a contentInset.

---

This is an internal issue. If you are a Googler, please visit [b/30021593](http://b/30021593) for more details.

---
#### Internal data
- Associated internal bug: [b/30021593](http://b/30021593)

**Comment by romoore** [comment]
Collections is not part of the accessibility leveling project.

**Comment by ianegordon** [comment]
We are not intending on any significant updates to the current Collections component. 😞
We are working on a much simpler [List component](https://github.com/material-components/material-components-ios/tree/develop/components/List). 😄

**Comment by material-mirror** [comment]
The internal issue [b/30021593](http://b/30021593) is now closed. This issue is being closed as a result.
